Makefile

```diff
@@ -56,9 +56,13 @@ test:
 
 .PHONY: fmt
 ## Auto-format source code and report code-style violations (lint)
-fmt:
+fmt: addheaders
 	./scripts/fmt.sh
 
+.PHONY: addheaders
+addheaders:
```

Comment:
maybe call it `add_license_headers` -- I am old and I remember "file headers" :-)

Reply:
sure, updated.